storage/innobase/row/row0import.cc

```diff
@@ -4154,31 +4176,41 @@ fil_tablespace_iterate(
 		iter.file_size = file_size;
 		iter.n_io_buffers = n_io_buffers;
 
+		size_t buf_size= (1 + iter.n_io_buffers)
+							<< srv_page_size_shift;
+
 		/* Add an extra page for compressed page scratch area. */
-		iter.io_buffer = static_cast<byte*>(
-			aligned_malloc((1 + iter.n_io_buffers)
-				       << srv_page_size_shift, srv_page_size));
-
-		iter.crypt_io_buffer = iter.crypt_data
-			? static_cast<byte*>(
-				aligned_malloc((1 + iter.n_io_buffers)
-					       << srv_page_size_shift,
-					       srv_page_size))
-			: NULL;
-
-		if (block->page.zip.ssize) {
+		iter.io_buffer= static_cast<byte *>(
+			aligned_malloc(buf_size, srv_page_size));
+
+		if (iter.crypt_data)
+		{
+			iter.crypt_io_buffer= static_cast<byte *>(
+				aligned_malloc(buf_size, srv_page_size));
+			iter.crypt_tmp_buffer= static_cast<byte *>(
+				aligned_malloc(buf_size, CPU_LEVEL1_DCACHE_LINESIZE));
+		}
+		else
+		{
+			iter.crypt_io_buffer= NULL;
+			iter.crypt_tmp_buffer= NULL;
+		}
+
+		if (block->page.zip.ssize)
+		{
 			ut_ad(iter.n_io_buffers == 1);
-			block->page.frame = iter.io_buffer;
-			block->page.zip.data = block->page.frame
-				+ srv_page_size;
+			block->page.frame= iter.io_buffer;
+			block->page.zip.data=block->page.frame + srv_page_size;
 		}
 
-		err = callback.run(iter, block);
+		err= callback.run(iter, block);
 
-		if (iter.crypt_data) {
+		if (iter.crypt_data)
+		{
```

Comment:
It is awkward, but there are two formatting styles in InnoDB code base. The old style (as in this function) uses TAB for indentation, K&R style placement of opening braces, and a space on both sides of assignment operators. The two formatting styles should not be mixed, and unnecessary changes to formatting should be avoided.

Unfortunately, the code base is not compatible with tools like `clang-format`.

Reply:
I believe I have corrected this now, but could use another look.

Perhaps you are already aware but if not - there's a recent feature in github where you can add a `.git-blame-ignore-revs` file and it will [hide those commits from the blame](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view). Useful for doing a one time whole-codebase formatting commit so you can enforce style in CI. Not sure how other tooling might interact, but it could be a path to reduce reviewer & editor burden of mixed styles in the future.

Anyway, I think I covered the necessary changes and have requested a re-review.